packages/react/src/components/ComboChartCard/__snapshots__/ComboChartCard.story.storyshot

```diff
@@ -693,9 +693,9 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT Exper
                             >
                               <span
                                 className=""
-                                title="10/28/2018 12:34"
+                                title="10/28/2018 07:34"
```

Comment:
This all looks great, but I am curious why these snapshots changed. I would have expected them to stay the same if it were a seamless replacement.